packages/core/src/textures/resources/ImageResource.js

```diff
@@ -142,9 +142,10 @@ export default class ImageResource extends BaseImageResource
      * Called when we need to convert image into BitmapImage.
      * Can be called multiple times, real promise is cached inside.
      *
+     * @param {PIXI.BaseTexture} baseTexture - BaseTexture for this resource
      * @returns {Promise} cached promise to fill that bitmap
      */
-    process()
+    process(baseTexture)
```

Comment:
Why did you add this argument? Doesn't look like it's used.

Reply:
baseTexture.premultiplyAlpha

Reply:
Missed that okay